gradle-plugin/doc/src/main/asciidoc/inc/_groovy-configuration.adoc

```diff
@@ -567,6 +567,11 @@ endif::[]
 
 | *serviceAccount*
 | Labels and annotations applied to `ServiceAccount` objects.
+ifeval::["{task-prefix}" == "oc"]
```

Comment:
I think this block would only show up in case of gradle plugins.

Reply:
Ignore, I'm wrong. I didn't see this is gradle specific documentation. This should be shifted to kit as well in future.